.storybook/themes/news.scss

```diff
@@ -466,6 +465,62 @@
 				),
 			),
 			blocks: (
+				simple-list: (
```

Comment:
I'm going to quickly reorder this. don't want to give some of my alphabetization advocates on my team any heartattacks

Reply:
updated and reordered after two tries haha